orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -72,11 +80,11 @@ def _load_torch_allocator_cpp_extension():
 
     return load_inline(name='inline_extension', cpp_sources=[torch_cuda_allocator_addresses_cpp_source],
                         functions=['cuda_caching_allocator_raw_alloc_address', 'cuda_caching_allocator_raw_delete_address'],
-                        verbose=True, with_cuda=True)
+                        verbose=verbosity < Verbosity.WARNING, with_cuda=True)
 
 class ORTModule(torch.nn.Module):
 
-    def __init__(self, module):
+    def __init__(self, module, verbosity = Verbosity.WARNING):
```

Comment:
It might be better hide this on an internal flag until the "advanced configuration" story is implemented.
Verbosity is more of a debug feature and adding it to a public ctor would give it too much of visibility. If we add all features/settings in the ctor, it will be bloated and users will start using and BC issue can arise when we refatcor

Reply:
ok sure. I think though that we should have a verbosity option in the advanced configuration when we get to it. I'll remove it from the constructor for now.